### ISSUE TITLE
Crash while loading map with previous version

### ISSUE THREAD

**Issue by Destroyerrrocket** [raiguard/RateCalculator#97]
Hi!
I got this error while loading a save with an older version:
```
The mod Rate Calculator (2.4.8) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event RateCalculator::on_init()
__RateCalculator__/scripts/global-data.lua:57: attempt to compare number with nil
stack traceback:
	__RateCalculator__/scripts/global-data.lua:57: in function 'build_entity_rates'
	__RateCalculator__/scripts/global-data.lua:13: in function 'init'
	__RateCalculator__/control.lua:48: in function <__RateCalculator__/control.lua:45>
```
I have tried removing the mod and installing it again, If there's anything you'd like me to try I'd love to help.
Thank you for this amazing mod!

**Comment by raiguard** [comment]
Please send me your save file so I can sync with your active mods.

**Comment by Destroyerrrocket** [comment]
Here you go! https://drive.google.com/file/d/1Zw2JqaUYEBlX2Rh0P-l5uP4hLiRByb9o/view?usp=sharing
Sorry for the delay

**Comment by raiguard** [comment]
Thanks for the save, this should be fixed for 3.0.

**Comment by Destroyerrrocket** [comment]
Thank you very much!!! :D
